dmc/deployments.rb

```diff
@@ -1,2 +1,22 @@
+require 'interchange'
+
+require_relative 'deployments/entities/deployment'
+
+require_relative 'deployments/serializers/deployment_serializer'
+
+require_relative 'deployments/forms/create_form'
+
+require_relative 'deployments/repositories/deployment_repository'
+require_relative 'deployments/repositories/deployment_repository/in_memory'
+require_relative 'deployments/repositories/deployment_repository/perpetuity'
+
+require_relative 'deployments/use_cases/create'
+require_relative 'deployments/use_cases/list'
+
 module Deployments
+  STATE         = [:pending, :in_progress, :success, :failed]
```

Comment:
Do you think the constants should be global at the modules instead of particular to the specific entity? If you have more than one entity in the deployment module (like the Task entity) and both are states, and they are different, how do we handle that? I think these constants should be a details of implementation of the entity, what's your opinion about that?